src/coreclr/jit/gentree.h

```diff
@@ -6687,6 +6691,31 @@ struct GenTreePutArgSplit : public GenTreePutArgStk
 };
 #endif // FEATURE_ARG_SPLIT
 
+#if defined(TARGET_WASM)
+struct GenTreePutArgType : public GenTreeOp
+{
+private:
+    CorInfoType          m_CorInfoType;
+    CORINFO_CLASS_HANDLE m_ClsHnd;
```

Comment:
```suggestion
    CORINFO_CLASS_HANDLE m_ClsHnd; // The struct handle if this is a struct, the pointee type if this is a pointer.
```
To document we treat pointers somewhat specially here.